openedx/features/course_experience/api/v1/urls.py

```diff
@@ -17,6 +21,11 @@
         reset_course_deadlines,
         name='course-experience-reset-course-deadlines'
     ),
+    re_path(
```

Comment:
Nit: There's no reason for this to be an `re_path` just use the standard `path` which.